Sources/MediaCategories/MediaCategoryViewController.swift

```diff
@@ -148,7 +148,6 @@ class VLCMediaCategoryViewController: UICollectionViewController, UICollectionVi
         collectionView?.dataSource = editing ? editController : self
         collectionView?.delegate = editing ? editController : self
 
-        editController.resetSelections()
```

Comment:
Is there a specific reason why this is removed?

It could be used directly inside the VC since it's a VC now.

Perhaps in `viewWillAppear`?

Reply:
it's removed because it's not needed. Every cell is unchecked on prepare for reuse

Reply:
In this case, it doesn't seem to reset.

![demo](https://i.gyazo.com/042f74c460af27394c4ae36563ae4274.mp4)

Reply:
oh you're right so the array needs to be cleared but resetAllVisibleCell is not needed because of the prepare for reuse but this is still needed